.github/workflows/ci.yml

```diff
@@ -47,6 +49,28 @@ jobs:
     - name: Test projects
       run: dotnet test Microsoft-Perf-Tools-Linux-Android.sln --no-restore --verbosity normal
 
+    - name: Install PluginTool
+      run: dotnet tool install --global Microsoft.Performance.Toolkit.Plugins.Cli --version 0.1.25-preview      
+
+    - name: Create PTIX
+      run: |
+        $dirs = @{
```

Comment:
These seems the same as below. Ideally we should check to see if the dirs can be shared between different run cmds